app/views/clients/files/_file_row.haml

```diff
@@ -27,16 +27,24 @@
   .row
     .col-3
       - if file.visible_in_window
-        %em This file is visible in the window
-      .jThumb.mt-2{data: {file: file.id}}
+        .font-italic.mb-2 This file is visible in the window
+      - if file.content_type == 'application/pdf' && file.content.present?
```

Comment:
It looks like `app/controllers/clients/files_controller.rb` defines `has_thumb` as `@file.content_type == 'image/jpeg'`.  And I think we've done similar throughout the site, so let's invert this and only show thumbs for jpegs.

Reply:
Okay, I should be able to remove the `has_thumb` route altogether then. I don't understand why we make an API call to find the content type of the file, when we already know it in the template 🤔  but this is the only place it's used, so I think I can safely remove it.